tests/sender/send_lookup.py

```diff
@@ -70,17 +77,19 @@ def test_ssl_lookup_new_line(self):
         lookup = Lookup(name=self.lookup_name, historic_tag=None, con=con)
         p_headers = Lookup.list_to_headers(["KEY", "HEX", "COLOR"], "KEY")
         lookup.send_control("START", p_headers, "INC")
-        if len(con.socket.recv(1000)) == 0:
+        if len(TestLookup.read(con, 1000)) == 0:
             raise Exception("Not msg sent!")
```

Comment:
Same here, I don't know if a more concrete exception applies along these lines.

Reply:
Yes, it is just a test case